src/server/api/lib/twilio.js

```diff
@@ -403,12 +410,126 @@ async function handleIncomingMessage(message) {
   }
 }
 
+/**
+ * Search for phone numbers available for purchase
+ */
+async function searchForAvailableNumbers(twilioInstance, areaCode, limit) {
+  const count = Math.min(limit, 30); // Twilio limit
+  return twilioInstance.availablePhoneNumbers("US").local.list({
```

Comment:
let's make this an environment variable `getConfig(COUNTRY_CODE, organization) || "US"`

Reply:
Can I use the PHONE_NUMBER_COUNTRY env var, which appears to be used for formatting, or is there a reason to introduce a new one?